__init__.py

```diff
@@ -23,27 +23,18 @@
 def get_version_string():
     return str(bl_info['version'][0]) + '.' + str(bl_info['version'][1]) + '.' + str(bl_info['version'][2])
 
-# Monkey patch to add gather_gltf_hook, must be removed once handled upstream https://github.com/KhronosGroup/glTF-Blender-IO/issues/1009
+# gather_gltf_hook does not expose the info we need, make a custom hook for now
+# ideally we can resolve this upstream somehow https://github.com/KhronosGroup/glTF-Blender-IO/issues/1009
 from io_scene_gltf2.blender.exp import gltf2_blender_export
 from io_scene_gltf2.io.exp.gltf2_io_user_extensions import export_user_extensions
 orig_gather_gltf = gltf2_blender_export.__gather_gltf
 def patched_gather_gltf(exporter, export_settings):
     orig_gather_gltf(exporter, export_settings)
-    export_user_extensions('gather_gltf_hook', export_settings, exporter._GlTF2Exporter__gltf)
+    export_user_extensions('hubs_gather_gltf_hook', export_settings, exporter._GlTF2Exporter__gltf)
     exporter._GlTF2Exporter__traverse(exporter._GlTF2Exporter__gltf.extensions)
 
-# Monkey patch to add gather_joint_hook, must be removed once handled upstream
```

Comment:
This is now merged upstream https://github.com/KhronosGroup/glTF-Blender-IO/pull/1021